fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +220,56 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+# If --dcmethod=NONE                                   # WARNING: The fMRIVolume pipeline is being run without appropriate distortion correction of the fMRI image. 
+                                                       # This is NOT RECOMMENDED under normal circumstances. We will attempt 6 DOF FreeSurfer BBR registration of the 
+                                                       # distorted fMRI to the undistorted T1w image. Distorted portions of the fMRI data will not align with the cortical ribbon. 
+                                                       # In HCP data 30% of the cortical surface will be misaligned by at least half cortical thickness and 10% of the cortical 
+                                                       # surface will be completely misaligned by a full cortical thickness. At a future time, we may be able to add support for 
+                                                       # fieldmap-less distortion correction; however, no extant software package or pipelines have successfully demonstrated 
+                                                       # clear improvement over no distortion correction, in the direction towards gold standard fieldmap-based correction.
```

Comment:
The comments here don't match the current `log_Warn` message of what we converged on.

Reply:
@mharms  is this correct:

```
# WARNING: The fMRIVolume pipeline is being run without appropriate distortion correction of the fMRI image. 
# This is NOT RECOMMENDED under normal circumstances. We will attempt 6 DOF FreeSurfer BBR registration of 
# the distorted fMRI to the undistorted T1w image. Distorted portions of the fMRI data will not align with 
# the cortical ribbon. In HCP data 30% of the cortical surface will be misaligned by at least half cortical 
# thickness and 10% of the cortical surface will be completely misaligned by a full cortical thickness. 
# At a future time, we may be able to add support for fieldmap-less distortion correction. At this time, 
# however, despite ongoing efforts, this problem is unsolved and no extant approach has been successfully 
# shown to demonstrate clear improvement according to the accuracy standards of HCP-Style data analysis when 
# compared to gold-standard fieldmap-based correction.
```

Reply:
Well, if that matches the text of the current `log_Warn`, then yes.

Reply:
It does.